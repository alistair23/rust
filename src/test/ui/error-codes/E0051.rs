#![feature(asm, naked_functions)]

#[naked] //~ ERROR E0051
#[no_mangle]
#[inline(never)]
pub extern "C" fn naked_test(_fubar: u64) {
    unsafe { asm!("int3") }
}

pub fn main() {
    naked_test(1);
}
