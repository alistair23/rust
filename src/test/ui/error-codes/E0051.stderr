error[E0051]: unable to use `naked` on function with inputs
  --> $DIR/E0051.rs:3:1
   |
LL | #[naked]
   | ^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0051`.
